### ISSUE TITLE
Problem serving /static/

### ISSUE THREAD

**Issue by vilvadot** [JulienHe/firebase-next-routes-example#2]
Thank you very much for this repo! I was having incredible problems deploying my nextjs app to firebase.

Have you had any problem serving /static/ folder within your app? It seems everything linking to /static/ is broken and all files URLs throw an error.

Thank you!

**Comment by AkshayDwarakanath** [comment]
I have the exact same problem. 
Did u figure it out??

**Comment by vilvadot** [comment]
No. Finally ditched firebase hosting as it was giving me way too many problems.
